ee/clickhouse/queries/paths/paths.py

```diff
@@ -88,6 +92,14 @@ def get_path_query_by_funnel(self, funnel_filter: Filter):
         {path_query}
         """
 
+    def _get_timestamp_step(self, funnel_filter: Filter) -> Optional[int]:
+        if self._filter.funnel_paths == FUNNEL_PATH_DROPOFF:
+            return 0
+        elif self._filter.funnel_paths == FUNNEL_PATH_JOURNEY:
+            return funnel_filter.funnel_step
```

Comment:
Maybe not for this PR: Thoughts on letting people choose any 2 steps of the funnel, between which to show paths?

Reply:
definitely